packages/ssi-service/src/feedback-panel/feedback.js

```diff
@@ -115,7 +115,7 @@ window.customElements.define( 'op-feedback', class extends HTMLElement {
     }`;
     const variables = {
       feedback: {
-        title: '',
+        title: `Feedback from ${user.rhatUUID} on ${ formData.namedItem( 'feedbackType' ).value || undefined }`,
```

Comment:
Can you move this piece of code to the backend?

So when the JIRA ticket is being created from the Resolver, just add this title template to the post API request.

Reply:
@riginoommen Adding that to this line would be better IMO: https://github.com/1-Platform/one-platform/blob/master/packages/feedback-service/src/resolver.ts#L162

Reply:
And maybe use the username instead of the rhUUID. That will make it human-readable.